app/src/app/navtitle/navtitle.component.sass

```diff
@@ -37,14 +37,30 @@ h2
   top: 3px
 
   @media (max-width: $breakpoint-tablet)
-    top: -3px
     font-size: 9pt
     font-weight: 800
 
 .long-title
   @media (max-width: $breakpoint-tablet)
     display: none
 
+#navitembig
+  @media (max-width: $breakpoint-tablet)
+    display: none
+
+#navitemsmall
+  @media (min-width: $breakpoint-tablet)
+    display: none
```

Comment:
I don't think you need two classes. Suppose you've only one called nav-link. Because the mat-menu element is **not inside** the nav element, you could say in CSS:
 - if the screen is narrow then do not display nav-link that are inside nav elements;
 - if the screen is wide then do not display nav-link that are inside mat-menu.

Reply:
mat-menu
  .nav-link
    @media (min-width: $breakpoint-tablet)
      display: none
  
nav
  .nav-link
    @media (max-width: $breakpoint-tablet)
      display: none

Reply:
Something like that ?

mat-menu
  .nav-link
    @media (min-width: $breakpoint-tablet)
      display: none
  
nav
  .nav-link
    @media (max-width: $breakpoint-tablet)
      display: none

Reply:
Yes, but using descendent combinators would be even better.